src/components/HeroDecorative/HeroDecorative.js

```diff
@@ -58,6 +59,26 @@ export class HeroDecorative {
       this.decorativeContainer.style.backgroundImage = this.images.small
     }
   }
+
+  setEventMatchMedia () {
+    const mediaLarge = window.matchMedia(this.width.large)
+    const mediaMedium = (window.matchMedia(this.width.mediumMin) && window.matchMedia(this.width.mediumMax))
+    const mediaSmall = (window.matchMedia(this.width.smallMin) && window.matchMedia(this.width.smallMax))
```

Comment:
With the way this is working right now, the result is not accurate. The next time you come to class please ask me since it is hard to explain here.